test/about/ledgerPanelAdvancedPanelTest.js

```diff
@@ -91,7 +91,7 @@ let generateAndSaveRecoveryFile = function (recoveryFilePath, passphrase) {
   let recoveryFileContents = ''
 
   if (typeof passphrase === 'string') {
-    const date = moment().format('L')
+    const date = format(new Date(), 'MM/DD/YYYY')
```

Comment:
same concern as above- although, I know this is just a test 😄

Reply:
yes same as above answer